components/bl-pdfViewer-component/src/index.js

```diff
@@ -52,8 +52,10 @@ export default function PdfViewer({ component, eventHandlers, elRef }) {
   const onPageLoadSuccess = () => {
     const spaceForControls = getBottomOffset(controlsRef.current) - getBottomOffset(pageRef);
 
-    pageRef.firstChild.style.height = `calc(${ height } - ${ spaceForControls }px)`;
-    pageRef.firstChild.style.width = width;
+    pageRef.firstChild.style.height = height ? `calc(${ height } - ${ spaceForControls }px)` : 'auto';
+    pageRef.firstChild.style.width = width || 'auto';
+
+    pageRef.firstChild.style.overflow = 'auto';
```

Comment:
https://react-pdf.org/advanced#page-wrapping

How about that?

Reply:
we use a slightly different library
https://github.com/wojtekmaj/react-pdf/tree/v4.x